.github/workflows/release.yml

```diff
@@ -39,11 +41,40 @@ jobs:
       - name: Build package
         run: python -m build
 
+      - name: Publish to Test PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1.5
+        if: ${{ contains(github.ref, '-rc') }}
+        with:
+          repository_url: https://test.pypi.org/legacy/
+          print_hash: true
+          skip_existing: true
+          verbose: true
+
+      - name: Publish to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1.5
+        if: ${{ !contains(github.ref, '-rc') }}
+        with:
+          print_hash: true
+
+      - name: Parse version and output PyPI URL
+        id: version
+        # run littlepay first, so `littlepay -v` doesn't create the config file (which outputs a message)
+        run: |
+          pip install -e .
+          littlepay
+          output=$(littlepay -v)
+          version=${output/littlepay /}
+          version=${version/.0/.}
+          echo "PYPI_RELEASE_URL=https://pypi.org/project/calitp-littlepay/$version" >> "$GITHUB_OUTPUT"
```

Comment:
Hmm so this writes the PyPI URL even for TestPyPI publishes / pre-releases? I guess that's OK? Feels a little strange and won't resolve to anything until the actual release is published right? 

With my [previous comment](https://github.com/cal-itp/littlepay/pull/31#discussion_r1521995096) I was actually just talking about the _name_ of the output variable, but still anticipating we would have the correct (TestPyPI / PyPI) URL in both cases.